WebDriverAgentLib/Utilities/FBXCodeCompatibility.m

```diff
@@ -43,3 +43,28 @@ + (instancetype)fb_applicationWithPID:(pid_t)processID
 
 @end
 
+static BOOL FBShouldUseFirstMatchSelector = NO;
+static dispatch_once_t onceFirstMatchToken;
+@implementation XCUIElementQuery (FBCompatibility)
+
+- (XCUIElement *)fb_firstMatch
+{
+  SEL firstMatchSelector = NSSelectorFromString(@"firstMatch");
```

Comment:
however XCTest headers prior to XCode9 don't contain such protocol and only have XCUIElementQuery